params/config.go

```diff
@@ -47,7 +47,11 @@ var (
 		EthTxTypeCompatibleBlock: big.NewInt(86816005),
 		MagmaCompatibleBlock:     big.NewInt(99841497),
 		KoreCompatibleBlock:      big.NewInt(0xffffffff),
-		DeriveShaImpl:            2,
+		KIP103: &KIP103Config{
+			Kip103CompatibleBlock: big.NewInt(0xffffffff),
+			Kip103ContractAddress: common.HexToAddress("0x0"),
+		},
```

Comment:
I personally prefer XCompatibleBlock and related fields on the same level, like [how geth defines its EIP150Hash](https://github.com/ethereum/go-ethereum/blob/v1.11.4/params/config.go#L64-L65). Having nested struct is[ kind of frustrating](https://github.com/klaytn/klaytn/blob/dev/governance/default.go#L1146-L1149).

Reply:
@yoomee1313 @ian0371 Please give your opinion. I think both ways are acceptable. 
Ollie's suggestion provides a simple and unified configuration format, but `Kip103ContractAddress` can not be nil and has 0x0 by default even though a network doesn't want to enable kip103.

The following is what ollie suggests: https://github.com/aidan-kwon/klaytn/compare/230301-rebalancing...aidan-kwon:klaytn:tmp-rebalancing?expand=1

Reply:
The right side of the diff (from the link) looks concise, so I prefer no nested struct.

Reply:
@aidan-kwon It seems more natural for me to declare the hard fork block number independently, like the ollie's suggest.